src/level/level_state.rs

```diff
@@ -246,3 +243,8 @@ impl<'a, 'b> State<CustomGameData<'a, 'b>, StateEvent> for LevelState {
         Trans::None
     }
 }
+
+fn do_test_method(data: StateData<CustomGameData>) {
+    let world = data.world;
+    LevelState::initialize_base(world);
+}
```

Comment:
i don't wanna do that, because the `handleEvent` is already long enough. there is no value in looking at an messy method, while most of the code has no other purpose than testing. its enough to know, that Space executes a testMethod.

Reply:
inlining this would add zero lines actually :) but anyway, it's not important.

Reply:
thanks you :)